Source/bmalloc/libpas/libpas.xcodeproj/project.pbxproj

```diff
@@ -1487,6 +1500,11 @@
 				0F59A399266844A300A38657 /* jit_heap_config.c */,
 				0F59A3902667F00E00A38657 /* jit_heap_config.h */,
 				0F59A3972668317400A38657 /* jit_heap_config_root_data.h */,
+				F30C5BE52E8F348F006CA1E9 /* mar_crash_reporter_report.h */,
+				F30C5BE72E8F348F006CA1E9 /* mar_registry.c */,
+				F30C5BE62E8F348F006CA1E9 /* mar_registry.h */,
+				F30C5BE92E8F348F006CA1E9 /* mar_report_crash.c */,
+				F30C5BE82E8F348F006CA1E9 /* mar_report_crash.h */,
```

Comment:
Let's rename them to `pas_mar_xxx` for libpas's naming consistency. If this part is not something on the top of libpas (I mean, if it is called in the core of libpas), then it should be named in that way.